seaclouds-on-byon.yaml

```diff
@@ -0,0 +1,63 @@
+name: SeaClouds platform
+
+location:
+  byon:
+    user: vagrant
+    privateKeyFile: ~/seaclouds-distribution/seaclouds_id_rsa
+    hosts:
+    - 192.168.100.10
+    - 192.168.100.11
+
+services:
+- serviceType: brooklyn.entity.basic.SameServerEntity
```

Comment:
`seaclouds.yaml` is not using `SameServerEntity` this is the only diff of these two yaml, it is just a simplification for the end-user not really needed